helm/ako/values.yaml

```diff
@@ -22,6 +22,8 @@ AKOSettings:
   namespaceSelector:
     labelKey: ""
     labelValue: ""
+  ServicesAPI: false # Flag that enables AKO in services API mode:https://kubernetes-sigs.github.io/service-apis/ . Currently implemented only for L4. This flag uses the upstream GA APIs which are not backward compatible 
```

Comment:
`servicesAPI` for keeping it consistent with other variables starting with small-case

Reply:
thx. done.